platform/Effect.roc

```diff
@@ -85,7 +85,7 @@ cwd : Effect (List U8)
 sendRequest : Box InternalHttp.Request -> Effect InternalHttp.InternalResponse
 
 tcpConnect : Str, U16 -> Effect (Result U64 Str)
-tcpClose : U64 -> Effect (Result {} *)
+tcpClose : U64 -> Effect {}
```

Comment:
See https://roc.zulipchat.com/#narrow/stream/231634-beginners/topic/error.20unification/near/448982717 and https://roc.zulipchat.com/#narrow/stream/231634-beginners/topic/error.20unification/near/449304004